.evergreen/generate_evergreen_tasks.js

```diff
@@ -247,6 +247,44 @@ TASKS.push(
   });
 });
 
+// Add task for testing lambda example without aws auth.
+TASKS.push({
+  name: 'test-lambda-example',
+  tags: ['latest', 'lambda'],
+  commands: [
+    { func: 'install dependencies' },
+    {
+      func: 'bootstrap mongo-orchestration',
+      vars: {
+        VERSION: 'latest',
+        TOPOLOGY: 'server'
+      }
```

Comment:
Should we be testing against the latest server or should we pin it?  I'm reluctant to test against latest for tasks that don't explicitly require it on the off chance that a server change breaks this test in the future.

Reply:
Switched to rapid.